packages/oas/src/utils/resolveRef.ts

```diff
@@ -0,0 +1,25 @@
+export const resolveRef = (
+  spec: Record<string, any>,
```

Comment:
Avoid using `any`, the preferred way is `unknown`

```suggestion
  spec: Record<string, unknown>,
```